.github/workflows/checking.yml

```diff
@@ -0,0 +1,13 @@
+name: Link Checking
+
+on:
+  schedule:
+    - cron: 0 0 * * * # At 00:00 (UTC)
```

Comment:
Nit picks: this could say "daily".

Reply:
https://github.com/tc39/proposals/pull/315#issuecomment-700347955

Reply:
This hides the key information

1. Run time 
2. Timezone

Reply:
I mean in addition to what you already have.

Reply:
@septs Does that info matter, though? Your choices of time and timezone were both arbitrary, no?

Reply:
This issue need a guideline.

Reply:
"@daily" is an alias for `0 0 * * * ` -- writing it out by hand seems really unnecessary. https://www.man7.org/linux/man-pages/man5/crontab.5.html#EXTENSIONS

Additionally, not using `@daily` makes it seem to the untrained eye that this is somehow special. That it must run at this time, and not any overrides that might be present on a host machine via anacrontab -- which means that the exact time is more important than the fact that it runs every day. That isn't the case here.

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule

Reply:
Ah, its not on that page.

Github actions do not support cron aliases: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#scheduled-events

So you are right, this has to be `0 0 * * *`. We can resolve this issue.

Reply:
Maybe add a comment to that effect in case anyone tries.

Something to the effect of `// Github actions does not support cron aliases` should be enough

Reply:
i think, this is unpredictable random event. is this really what we need?

Reply:
What you have looks good -- we can't use `daily` as per the documentation you linked. We just need the comment, because people are likely to get confused (as I did) about why we _aren't_ using `daily`. Someone might try to change it as a "fix", and might end up inadvertently breaking the cron job. We don't want that. 

Outside of github actions, sometimes you want to override cron jobs, so `daily` acts like a variable that you can set to a specific time for multiple scripts if necessary. It is also more readable, and unless there is an override it usually means `0 0 * * *`.

Reply:
We maybe need use **GitHub Support**.